forte/data/data_store.py

```diff
@@ -250,6 +251,44 @@ def _new_group(self, type_name: str, member_type: str) -> List:
         entry += len(self._type_attributes[type_name]) * [None]
         return entry
 
+    def _is_subclass(self, type_name: str, cls) -> bool:
+        r"""This function takes a fully qualified ``type_name`` class name,
+        ``cls`` class and returns whether ``type_name``  class is the``cls``
+        subclass or not.
+
+        Args:
+            type_name: A fully qualified name of an entry class
+            cls: An entry class.
+
+        Returns:
+            A boolean value whether ``type_name``  class is the``cls``
+            subclass or not.
+
+        """
+        if type_name not in self._type_attributes:
+            self._type_attributes[type_name] = {}
+        if "parent_class" not in self._type_attributes[type_name]:
+            self._type_attributes[type_name].update(parent_class=[])
```

Comment:
why using `update` instead of simply `["parent_class"]=[]`

Reply:
OK, will change in the next PR.